js/base/Exchange.js

```diff
@@ -2361,36 +2361,33 @@ module.exports = class Exchange {
         return this.filterBySymbolSinceLimit (sorted, symbol, since, limit);
     }
 
-    isPostOnly (type, timeInForce = undefined, exchangeSpecificOption = undefined, params = {}) {
+    isPostOnly (type, timeInForce, params = {}) {
```

Comment:
The exchangeSpecificOption is important, sometimes an excahgne will have something like `"post_only": true` and we need to check for that as well